src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -158,6 +158,10 @@ private function doAsk(OutputInterface $output, Question $question)
         $inputStream = $this->inputStream ?: STDIN;
         $autocomplete = $question->getAutocompleterValues();
 
+        if (\function_exists('sapi_windows_cp_set')) {
+            sapi_windows_cp_get(437);
```

Comment:
Is it possible to add a comment to document what 437 means?

Reply:
👍 fixed